messages/add.md

```diff
@@ -1,34 +1,30 @@
 # commandSummary
 
-add custom rules to Salesforce Code Analyzer's registry
+Add custom rules to Salesforce Code Analyzer's registry to run them along with the built-in rules.
 
 # commandDescription
 
-Adds custom rules to Salesforce Code Analyzer's registry so that you can run them along with the built-in rules. Compile and test custom rules separately before adding them.
+Compile and test custom rules separately before adding them.
```

Comment:
I see you lengthened our summaries and shortened our descriptions. Is it normal that the descriptions contain just supplemental information to the summaries instead of full information?